Changelog.md

```diff
@@ -8,6 +8,7 @@ Compiler Features:
  * SMTChecker: Support events and low-level logs.
  * SMTChecker: Support ``revert()``.
  * SMTChecker: Support shifts.
+ * SMTChecker: Support compound and, or, and xor operators.
```

Comment:
And shifts?

Reply:
I was thinking that it is not needed, as in the same release we added support for shifts. Will add one as probably it is still fine to have both shifts and compound shifts in same release.

Reply:
Ah ok, makes sense

Reply:
Ok, let's than add changelog item if PR doesn't make this release?

Reply:
Yep!